torch/csrc/jit/codegen/cuda/glfdc/bitvector.cpp

```diff
@@ -0,0 +1 @@
+#include <torch/csrc/jit/codegen/cuda/glfdc/bitvector.h>
```

Comment:
Do we need this file?

Reply:
Not really, beyond checking header compiles standalone.

Reply:
I'd also prefer not having these cpps to save some object files in the build : ).